sdk/metricsadvisor/ai-metrics-advisor/test/public/keyRotation.spec.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+
+import { MetricsAdvisorKeyCredential } from "../../src";
+import { testEnv } from "./util/recordedClients";
+
+describe("MetricsAdvisorKeyCredential", () => {
+  let credential: MetricsAdvisorKeyCredential;
+
+  beforeEach(function() {
+    // eslint-disable-next-line no-invalid-this
```

Comment:
Is this line needed? there's no `this` in the next line

Reply:
```suggestion
```